### DISCUSSION TITLE
Secondary Audio Tracks added to Movies

### DISCUSSION THREAD

**Discussion by Chapster5** [LizardByte/community#559] [Themerr]
### Select Themerr Product

Themerr-jellyfin

### Select Topic Area

Bug

### Body

Hi there,

I use themerr for my Jellyfin setup it gives a nice little extra to the viewing of things. I have noticed with Movies after themerr has indexed one when you play it. The default audio track of the movie is set to the themerr track rather than the movies originial audio. Have I got a setting wrong somewhere? How do I stop the movie from defaulting to the themerr track?

Thank you!

**Comment by ReenigneArcher** [comment]
Sounds like a Jellyfin bug. All Themerr did was provide a theme.mp3 file.

**Reply by Chapster5** [reply]
Ahh I see,

Where does it store the file? In with the media?

**Reply by ReenigneArcher** [reply]
Next to the movie/show.

**Reply by Chapster5** [reply]
Ahh yes so it does.

Think it is a Jellyfin issue the Themerr mp3 file even has -external on the end of it and the original audio track in the movie file says -default which you would think would be the first thing Jellyfin plays.

**Comment by ReenigneArcher** [comment]
Closing since Themerr is now archived.
